edge-util/test/Microsoft.Azure.Devices.Edge.Storage.RocksDb.Test/disk/DiskSpaceCheckerTest.cs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Storage.RocksDb.Test.Disk
+{
+    using System;
+    using System.IO;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Storage.RocksDb.Disk;
+    using Microsoft.Azure.Devices.Edge.Util.Test.Common;
+    using Xunit;
+
+    [Unit]
+    public class DiskSpaceCheckerTest
```

Comment:
>DiskSpaceCheckerTest [](start = 17, length = 20)

nit: rename to StorageSpaceCheckerTest

Reply:
Leaving this one alone since it's a specific test for DiskSpaceChecker